src/elexmodel/handlers/data/VersionedData.py

```diff
@@ -143,15 +147,18 @@ def compute_estimated_margin(df):
             # Compute batch_margin using NumPy
             # this is the difference in dem_votes - the difference in gop_votes divided by the difference in total votes
             # that is, this is the normalized margin in the batch of votes recorded between versions
-            batch_margin = (
-                np.diff(results_dem, append=results_dem[-1]) - np.diff(results_gop, append=results_gop[-1])
-            ) / np.diff(results_weights, append=results_weights[-1])
+            with warnings.catch_warnings():
```

Comment:
where is this coming from?

Reply:
It happens if you have any zeros in the denominator.  Since we probably won't enable this until late into tab anyway, it should be pretty rare, but still, it's nice to not see it in our logs if we know it's a thing we're ok with 🤔

Reply:
I just realized saying "it happens if you have zeros in the denominator" is not the most useful comment here lol 😅  More like, if you have any `results_weights` that are still zero due to units that haven't reported, especially in states with a lot of small units (New England, Texas) 🤔